projects/v3/src/app/components/assessment/assessment.component.ts

```diff
@@ -95,6 +96,22 @@ export class AssessmentComponent implements OnChanges {
     this.isPendingReview = false;
   }
 
+  /**
+   * this event will update saving message when saving in progress and done.
+   * this event is broadcasting from activity-desktop and assessment-mobile components.
+   */
+  private _initSavingEvent() {
```

Comment:
This `assessment.component` is a stand-alone component, which shouldn't subscribe to a global wide event.
Why do we need to check the saving message outside of this component?